.buildkite/pipeline.yml

```diff
@@ -55,19 +80,18 @@ steps:
         account_ids: "436617320021"
         no-include-email: true
         region: "us-west-2"
-      docker#v2.1.0:
+      docker#v3.2.0:
+        shell: ["/bin/bash", "-i", "-e", "-c"]
         debug: true
-        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:centos7_2-3"
+        image: "436617320021.dkr.ecr.us-west-2.amazonaws.com/ci:centos7_2-4"
         propagate-environment: true
         workdir: /data/job
     timeout: 120
 
-  - command: | # Ubuntu 16.04 Build
-        echo "+++ :hammer: Building"
-        ./scripts/eosio_build.sh -y -P -m
-        echo "--- :compression: Compressing build directory"
```

Comment:
Please don't delete these types of print statements.
Without this print statement, it will look like the agent has hanged.